src/etc/rc.filter_synchronize

```diff
@@ -266,10 +266,16 @@ if (is_array($config['hasync'])) {
 		}
 		$sections[] = 'wol';
 	}
-	if ($hasync['synchronizetrafficshaper'] != "" and is_array($config['shaper'])) {
+	if ($hasync['synchronizetrafficshaper'] != "") {
+		if (!is_array($config['shaper'])) {
+			$config['shaper'] = array();
+		}
 		$sections[] = 'shaper';
 	}
-	if ($hasync['synchronizetrafficshaperlimiter'] != "" and is_array($config['dnshaper'])) {
+	if ($hasync['synchronizetrafficshaperlimiter'] != "") {
+		if (!is_array($config['dnshaper'])) {
```

Comment:
See above note, same can be done here (but for `dnshaper`)

Reply:
done